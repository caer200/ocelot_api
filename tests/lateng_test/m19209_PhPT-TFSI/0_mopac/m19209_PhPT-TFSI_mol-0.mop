CHARGES

 
H	6.411584 +1 6.279214 +1 4.507980 +1
C	6.463253 +1 6.160689 +1 5.449214 +1
C	6.533534 +1 7.289884 +1 6.292076 +1
C	6.468647 +1 4.892045 +1 5.975117 +1
C	6.631695 +1 7.075735 +1 7.685332 +1
N	6.490055 +1 8.562307 +1 5.742735 +1
H	6.411451 +1 4.142460 +1 5.394191 +1
C	6.557846 +1 4.690745 +1 7.361218 +1
C	6.641071 +1 5.768292 +1 8.199431 +1
S	6.736079 +1 8.343875 +1 8.835696 +1
C	6.564982 +1 9.744661 +1 6.463417 +1
C	6.242205 +1 8.661445 +1 4.310504 +1
H	6.560310 +1 3.809484 +1 7.715889 +1
H	6.705875 +1 5.631501 +1 9.137198 +1
C	6.685297 +1 9.762549 +1 7.871329 +1
C	6.508641 +1 10.982191 +1 5.790562 +1
C	7.311849 +1 8.779101 +1 3.441429 +1
C	4.932262 +1 8.634362 +1 3.871011 +1
C	6.752425 +1 10.982317 +1 8.560656 +1
H	6.429490 +1 10.997886 +1 4.844073 +1
C	6.566642 +1 12.164797 +1 6.485119 +1
H	8.204630 +1 8.778761 +1 3.766291 +1
C	7.049230 +1 8.898017 +1 2.079623 +1
H	4.214345 +1 8.543336 +1 4.486406 +1
C	4.690470 +1 8.743452 +1 2.505492 +1
H	6.845358 +1 10.984181 +1 9.506067 +1
C	6.685214 +1 12.172985 +1 7.881822 +1
H	6.525826 +1 12.988187 +1 6.013039 +1
H	7.767463 +1 8.986297 +1 1.464186 +1
C	5.744017 +1 8.887813 +1 1.620934 +1
H	3.798569 +1 8.718749 +1 2.179186 +1
H	6.718795 +1 12.996249 +1 8.354664 +1
H	5.570124 +1 8.980628 +1 0.691602 +1
