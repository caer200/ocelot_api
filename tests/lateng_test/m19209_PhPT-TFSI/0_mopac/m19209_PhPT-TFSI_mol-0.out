 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:29:53 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       H          6.41158400  *   6.27921400  *   4.50798000  *
     2       C          6.46325300  *   6.16068900  *   5.44921400  *
     3       C          6.53353400  *   7.28988400  *   6.29207600  *
     4       C          6.46864700  *   4.89204500  *   5.97511700  *
     5       C          6.63169500  *   7.07573500  *   7.68533200  *
     6       N          6.49005500  *   8.56230700  *   5.74273500  *
     7       H          6.41145100  *   4.14246000  *   5.39419100  *
     8       C          6.55784600  *   4.69074500  *   7.36121800  *
     9       C          6.64107100  *   5.76829200  *   8.19943100  *
    10       S          6.73607900  *   8.34387500  *   8.83569600  *
    11       C          6.56498200  *   9.74466100  *   6.46341700  *
    12       C          6.24220500  *   8.66144500  *   4.31050400  *
    13       H          6.56031000  *   3.80948400  *   7.71588900  *
    14       H          6.70587500  *   5.63150100  *   9.13719800  *
    15       C          6.68529700  *   9.76254900  *   7.87132900  *
    16       C          6.50864100  *  10.98219100  *   5.79056200  *
    17       C          7.31184900  *   8.77910100  *   3.44142900  *
    18       C          4.93226200  *   8.63436200  *   3.87101100  *
    19       C          6.75242500  *  10.98231700  *   8.56065600  *
    20       H          6.42949000  *  10.99788600  *   4.84407300  *
    21       C          6.56664200  *  12.16479700  *   6.48511900  *
    22       H          8.20463000  *   8.77876100  *   3.76629100  *
    23       C          7.04923000  *   8.89801700  *   2.07962300  *
    24       H          4.21434500  *   8.54333600  *   4.48640600  *
    25       C          4.69047000  *   8.74345200  *   2.50549200  *
    26       H          6.84535800  *  10.98418100  *   9.50606700  *
    27       C          6.68521400  *  12.17298500  *   7.88182200  *
    28       H          6.52582600  *  12.98818700  *   6.01303900  *
    29       H          7.76746300  *   8.98629700  *   1.46418600  *
    30       C          5.74401700  *   8.88781300  *   1.62093400  *
    31       H          3.79856900  *   8.71874900  *   2.17918600  *
    32       H          6.71879500  *  12.99624900  *   8.35466400  *
    33       H          5.57012400  *   8.98062800  *   0.69160200  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         H          6.4116      6.2792      4.5080
     2         C          6.4633      6.1607      5.4492
     3         C          6.5335      7.2899      6.2921
     4         C          6.4686      4.8920      5.9751
     5         C          6.6317      7.0757      7.6853
     6         N          6.4901      8.5623      5.7427
     7         H          6.4115      4.1425      5.3942
     8         C          6.5578      4.6907      7.3612
     9         C          6.6411      5.7683      8.1994
    10         S          6.7361      8.3439      8.8357
    11         C          6.5650      9.7447      6.4634
    12         C          6.2422      8.6614      4.3105
    13         H          6.5603      3.8095      7.7159
    14         H          6.7059      5.6315      9.1372
    15         C          6.6853      9.7625      7.8713
    16         C          6.5086     10.9822      5.7906
    17         C          7.3118      8.7791      3.4414
    18         C          4.9323      8.6344      3.8710
    19         C          6.7524     10.9823      8.5607
    20         H          6.4295     10.9979      4.8441
    21         C          6.5666     12.1648      6.4851
    22         H          8.2046      8.7788      3.7663
    23         C          7.0492      8.8980      2.0796
    24         H          4.2143      8.5433      4.4864
    25         C          4.6905      8.7435      2.5055
    26         H          6.8454     10.9842      9.5061
    27         C          6.6852     12.1730      7.8818
    28         H          6.5258     12.9882      6.0130
    29         H          7.7675      8.9863      1.4642
    30         C          5.7440      8.8878      1.6209
    31         H          3.7986      8.7187      2.1792
    32         H          6.7188     12.9962      8.3547
    33         H          5.5701      8.9806      0.6916

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: C18 H13 N S  =    33 atoms



      MOLECULAR POINT GROUP   :   C1  

          COMPUTED CHARGE ON SYSTEM: +0

   COMPUTED CHARGE ON SYSTEM = +0. THIS AGREES WITH THE CHARGE IN THE DATA-SET.

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
